picard/ui/mainwindow.py

```diff
@@ -836,6 +836,10 @@ def view_info(self):
         else:
             file = self.tagger.get_files_from_objects(self.selected_objects)[0]
             dialog = FileInfoDialog(file, self)
+        try:
+            dialog.ui.tabWidget.setCurrentIndex(default_tab)
+        except IndexError:
```

Comment:
Under which circumstances does the `setCurrentIndex` call above raise an `IndexError`? It doesn't do it for 50000 and -5.